tests/unit/components/ObsDetails/ObsDetails.test.js

```diff
@@ -73,15 +73,6 @@ jest.mock( "components/ObsDetails/DataTab", () => ( {
   default: () => mockDataTab
 } ) );
 
-jest.mock(
```

Comment:
I have added those mocks to make the unit tests of these components more granular, that they don't care what happens in the container and only in the component we want to test.

I am not sure how these mocks can be the source of the footer showing icon-related warnings. I put them in place extra to not have the footer in the element tree. Curious, how did it help you to remove those mocks?

Reply:
Whoops good catch -- just me going too quickly trying to eliminate causes. I added these back and will merge.